### PR TITLE
Impl oidcspa

### PR DESCRIPTION
**Pull request by EricThuaud**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # REACT_APP_AUTH_TYPE: oidc / none, default: none
```
Comment (SimonDmz) [inline .env:1]: should change default value in comment line ?
 none => oidc

File: src/functions/oidc.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { createMockReactOidc } from "oidc-spa/mock/react";
+import { createReactOidc } from "oidc-spa/react";
+import { environment, oidcConf } from "utils/read-env-vars";
+
+const guestUser = {
+  inseegroupedefaut: ["utilisateur_Platine"],
```
Comment (SimonDmz) [inline src/functions/oidc.js:6]: 👮‍♂️ shoudn't expose/hardcode internal security groups
=> could be defined in app/oidc configuration

File: src/ui/context/UserAccount/index.jsx [hunk 3]
```diff
@@ -86,8 +85,8 @@ export const UserAccountProvider = ({ children }) => {
      * Please change in ui/context/auth/provider/NoAuth.jsx, "oidcUser: { id: id }," to "`oidcUser: { preferred_username: id },"
```
Comment (SimonDmz) [inline src/ui/context/UserAccount/index.jsx:85]: if changes have been made => just keep `For Keyloack : check if the id is "id" or something like : "preferred_username"` in the comment

File: src/functions/autoLogoutCountdown.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        alignItems: "center",
+        textAlign: "center",
+        color: "white",
+        fontWeight: "bold",
+        zIndex: 1200,
+      }}
+    >
+      <div>
+        <Typography variant="h5">{"Vous êtes toujours là?"}</Typography>
+        <Typography>{`Vous allez être déconnecté(e) dans ${secondsLeft} sec.`}</Typography>
```
Comment (SimonDmz) [inline src/functions/autoLogoutCountdown.js:61]: dictionary ?

File: .env (hunk 1 above)
Comment (EricThuaud) [reply .env:1]: done

File: src/functions/oidc.js (hunk 2 above)
Comment (EricThuaud) [reply src/functions/oidc.js:6]: done with env var implem

File: src/ui/context/UserAccount/index.jsx (hunk 3 above)
Comment (EricThuaud) [reply src/ui/context/UserAccount/index.jsx:85]: comment now useless,removed

File: src/functions/autoLogoutCountdown.js (hunk 4 above)
Comment (EricThuaud) [reply src/functions/autoLogoutCountdown.js]: done

**Pull request merged**